README.md

```diff
@@ -20,3 +20,36 @@ To run the app you should:
 4. Run the app with `bin/rails s`
 
 To check everything is working OK run the app and check that the navbar appears correctly and that the dropdown works.
+
+
+## 2. Exam instructions
+
+The world famous Marie Kondo, known for being an organizing consultant, needs the heklp of the best Software Engineers in the planet: students of the Web Technologies in Universidad de los Andes. She wants to build a web application to keep the stock of garments her clients have in their closets.
+
```

Comment:
`The world famous Marie Kondo, known for being an organizing consultant, needs the heklp of the best Software Engineers in the planet: students of the Web Technologies in Universidad de los Andes. She wants to build a web application to keep the stock of garments her clients have in their closets.`

The world famous Marie Kondo, known for being an organizing consultant, needs the help of the best Software Engineers in the planet: students of the Web Technologies in Universidad de los Andes. She wants to build a web application to keep the stock of garments her clients have in their closets.